packages/mrm/src/index.js

```diff
@@ -362,10 +364,77 @@ function tryFile(directories, filename) {
  * Try to resolve any of the given npm modules. Works with local files, local and global npm modules.
  *
  * @param {string[]} names...
- * @return {any}
+ * @return {Promise<any>}
+ */
+async function tryResolve(...names) {
+	const result = firstResult(names, requireg.resolve);
+	if (result) {
+		return result;
+	}
+
+	const possibleGlobals = await Promise.all(
+		names.map(name => packageJson(name).catch(() => {}))
+	);
+	/**
+	 * @type packageJson.AbbreviatedMetadata[]
+	 */
+	// @ts-ignore
+	const choices = possibleGlobals.filter(Boolean);
+
+	let questions;
+	if (choices.length === 1) {
+		// @ts-ignore
+		const pkgName = choices[0].name;
+		questions = [
+			{
+				name: 'pkgName',
+				message: `Found a package named ${pkgName} on NPM would you like to install it globally?`,
+				type: 'confirm',
+			},
+		];
+	} else {
+		questions = [
+			{
+				name: 'pkgName',
+				message: `Which package would you like to install globally?`,
+				// @ts-ignore
+				choices: choices
+					.map(({ name }) => ({ name, value: name }))
+					.concat({
+						name: 'I do not want to install any these packages',
+						value: false,
+					}),
+				type: 'list',
+			},
+		];
+	}
+	let { pkgName } = await inquirer.prompt(questions);
```

Comment:
We could move the package selection to it's own function to avoid reassignment.

Reply:
@sapegin I ended up keeping it within this function because coupled with your last suggestion I was able to simplify the prompt further